docs/overview/roadmap.md

```diff
@@ -104,7 +104,15 @@ the maintainers’ radar, with clear interest expressed by the community.
 - **Security Plan (and Strategy):** The purpose of the Security Strategy is to
   move another step along the path to maturing the platform, setting the
   expectations of any adopters from a security standpoint.
-- **Search GA:**.
+- **Search 1.0:** Fix the few remaining issues to get Backstage Search platform
```

Comment:
could we link to the "few remaining issues" here to help the reader?

Reply:
some alternatives depending on what its referring to 
All issues labeled with search: https://github.com/backstage/backstage/issues?q=is%3Aissue+is%3Aopen+label%3Asearch

Search milestones:
https://github.com/backstage/backstage/milestone/27
https://github.com/backstage/backstage/milestone/28